Dockerfile

```diff
@@ -10,11 +10,7 @@ ENV SCRIPT_BASE=./contrib/cirrus
 ENV PACKER_BASE=$SCRIPT_BASE/packer
 
 # Only add minimal tooling necessary to complete setup.
```

Comment:
@cevich This comment no longer seems accurate after the change. Is adding everything to the container from the libpod directory really the "minimal tooling to necessary complete setup"? On my checkout of `libpod` I have 126 MB in the current directory and running `make test` visibly stalled at this step for so long that I stopped to investigate.

For contrast, I ran this on the old "ADD" commands:

      du -hc ./contrib/cirrus/ ./hack/install_catatonit.sh cni/*.conflist test/*.json test/*.conf

That totaled 304K.

Did the minimal files needed for tooling really increase from 304K to 126M?

Reply:
Sorry I'm late.  You're right, it's not an accurate comment.  At one point I was much more selective in the Dockerfile about only adding what was needed, but...kept missing stuff.  So I just tossed it all in and forgot about it :grin: I'll open another PR to fix this.

Reply:
#6904